scalableminds/webknossos
# Fix nightly (fix tokens, upgrade puppeteer)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- see ci run of nightly https://app.circleci.com/jobs/github/scalableminds/webknossos/16878?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

### Issues:
- fixes correct dev installment during nightly
- upgrades puppeteer
- fixes token

------
(Please delete unneeded items, merge only when none are left open)
